### DISCUSSION TITLE
How to use virtio-net to configure the network interface in cloud-hypervisor? Are there any reference examples?

### DISCUSSION THREAD

**Discussion by lebron8dong** [cloud-hypervisor/cloud-hypervisor#6412] [Q&A]
**Describe the bug**
There is no network in the cloud-hypervisor virtual machine

**To Reproduce**
Tutorial：Getting started - Firmware Booting

**Version**
Output of `cloud-hypervisor --version`:
cloud-hypervisor v37.0-56-g6aa7afbb

I want to run resnet50 in cloud-hypervisor. I need to download the model and dependencies. How to configure the network?

**Comment by up2wing** [comment]
See docs/macvtap-bridge.md please ：）
